### ISSUE TITLE
Temp removing Telerik and benetskyybogdan.medium.com

### ISSUE THREAD

**Issue by jamesmontemagno** [planetxamarin/planetxamarin#708]
These blogs have some odd issue where they keep popping up to the top and also keep getting tweeted out daily https://twitter.com/planetxamarin

**Comment by bbenetskyy** [comment]
@jamesmontemagno what should I do to have me back to this list? Same problem I see here with other tweets as well:


![image](https://user-images.githubusercontent.com/8330262/106556029-fe9c1c00-651e-11eb-89a7-9462c2478cad.png)

**Comment by bbenetskyy** [comment]
@jamesmontemagno also other members from medium feed works well and was not disabled, maybe it was a temporary issue and can try to get it back?

**Comment by jamesmontemagno** [comment]
I am curious if it is something else. super weird.....

**Comment by jamesmontemagno** [comment]
@Cheesebaron maybe some ids or dates are odd in the rss and some posts are being triggered as new multiple times.... idk odd

**Comment by Cheesebaron** [comment]
I don't see any dupes in our feed?

**Comment by bbenetskyy** [comment]
In a couple of days, I plan to add one more blog post and we will see if that bug still exist and what's actually happens.

Before publishing the blog post I left you guys to know about that here

**Comment by jamesmontemagno** [comment]
@Cheesebaron it's not in the feed, checkout https://twitter.com/planetxamarin which is basically looking at changes in the RSS via IFTTT

**Comment by Cheesebaron** [comment]
@jamesmontemagno how does this IFTTT thing detect uniqueness of items?

**Comment by jamesmontemagno** [comment]
https://ifttt.com/feed ? not sure...

**Comment by jamesmontemagno** [comment]
I guess we could try to hook up a logic app or something else to it to try it out to see if there are duplicates that come through.

**Comment by Cheesebaron** [comment]
It is probably based on the URL. Looking at the twitter account, it seems to be related to https://luismts.com. One thing the author does is to post in multiple languages, maybe something is up with that.

@jamesmontemagno which URL are you using for the IFTTT? The feed url that only grabs English blogs?

**Comment by jamesmontemagno** [comment]
Yeah, I am using -> http://www.planetxamarin.com/feed?lang=en

**Comment by jamesmontemagno** [comment]
Yeah, it does look like that is the issue.... Let me try the normal feed maybe.

**Comment by luismts** [comment]
Hi, I'm seeing my name and blog here. 

Yes, that behavior was a little odd. But it happened with other posts too.

I'm looking at what is happening with my English feed.  Right now just Spanish posts are being published.  

Do you know what is happening? 

cc @Cheesebaron  @jamesmontemagno

**Comment by bbenetskyy** [comment]
Yesterday I published a new blog post. At `https://twitter.com/PlanetXamarin` it appears in ~2h as only **one tweet**. 

Today I try to edit my blog post - because I thought that updates my cause new tweets, but in 2 hours after update a blog post there still no new tweets at tweeter.

I think problem is fixed and all works as expected 😊
